src/mongo_log.c

```diff
@@ -32,10 +33,18 @@ void mongolite_log_handler (mongoc_log_level_t event, const char *log_domain, co
 
 void R_init_mongolite(DllInfo *info) {
   static mongoc_log_func_t logfun = mongolite_log_handler;
+  char *r_version;
   mongoc_init();
+#if defined (R_VERSION)
```

Comment:
The R manual section 6.14 references [`R_VERSION`](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Platform-and-version-information).